### ISSUE TITLE
Question about online/in-person workshop asked twice 

### ISSUE THREAD

**Issue by maneesha** [carpentries/amy#2588]
We ask about whether the workshop will be in-person or online twice.  First we ask explicitly:

![image](https://github.com/carpentries/amy/assets/829690/17ade640-6afa-4b12-b927-74d7082bfbd5)

then we ask again as part of a "other info" question:

![image](https://github.com/carpentries/amy/assets/829690/ca463362-e09c-4d23-a78b-f0c24d1eb493)

I recommend changing the second question to read just `Is there any other information you would like to share with us?` and removing the help text `Knowing if this workshop is on-line or in-person will help ensure we can best support you in coordinating the event.`

Tagging @sheraaronhurt @danielle06 @elletjies  for feedback.

**Comment by sheraaronhurt** [comment]
@maneesha agree. We can keep it simple and ask **Is there any other information you would like to share with us?** 

Thanks!

**Comment by danielle06** [comment]
I agree as well, thanks @maneesha !
